### ISSUE TITLE
BUG - All - Sev 2 - VoiceOver doesn't announce the selected "To (Required)" recipient, no Done button on the selector

### ISSUE THREAD

**Issue by artsymartha68** [department-of-veterans-affairs/va-mobile-app#11453]
## Summary of the bug 
I received this bug report from a blind Veteran. She tried to compose a secure message. All the fields announced as expected except for the "To (Required)" field. She successfully sent a message but had no idea which recipient would receive it. 

## Steps to Reproduce the Bug 
1. Go to Messages.
2. Turn on VoiceOver. 
3. Select "Start a new message" button.
4. The "Pick a care system" selector operates as expected. There is a Done button on that selector. 
5. Now the To field displays. Swipe right to get to the next To field.
6. VO says "Select recipient. Open com-bah-box selector to select recipient." {NOTE: can we make "combobox" into two words, if it must read out? Or improve the announcement???}
7. Swipe past the search field. I can hear the various options read out as I have the To selector open. Choose a recipient and close the selector window by double tapping. There is no expected Done button in the selector as there is for the other fields. 
8. Once I choose a recipient from the list, then it never reads out my selection. Again, the other fields are just fine and announce properly as I swipe through them.

## Resulting behavior
What is the current behavior / experience? 

I can successfully send the message but I have no way of hearing which recipient I chose in the To field.

## Expected behavior
What did you expect to happen? 

1. She expected a Done button in the selector as in the other fields on the page
2. She expected the To recipient she chose to announce out loud so she knew who was getting her message.
3. Can we improve the readout of "cam-BAH-box"? Maybe at least two words? Combo box? Or just say select? 

## System Information:
- Apple/Android/Other: Confirmed on iPhone and Android
- Mobile Device: iPhone 13, latest build from today Aug 12
- Operating System: iOS 18.6
- App Version: 2.56.0 (2886)
- User Account: Demo mode
- Accessibility State: VoiceOver

## Screen shot, recording or additional information
Add any screen shots, video, gifs, json etc that will help the engineers track down the issue. Do not include any personally identifiable information for any real person, including screenshots.

**Comment by BeccaTupaj** [comment]
@digitalken Please test and see if this is happening on Android as well as iPhone. Please also assess the severity. 
cc: @timwright12

**Comment by mmutia** [comment]
Hi @BeccaTupaj and @digitalken,

I have confirmed that the bug also occurs on Android as well! Upon choosing a recipient from the "To" list, the selection cannot announced via TalkBack.

Additionally, the following buttons are unlabeled and require descriptive labels to inform the user of its purpose:

- the button to clear the "To" field
- the button to close the "Search for a VA health care team" modal

CC @artsymartha68

**Comment by digitalken** [comment]
PASS - iOS and Android look and sound good.
